elixir/README.md

```diff
@@ -300,7 +300,7 @@ Interactive Elixir (1.15.2) - press Ctrl+C to exit (type h() ENTER for help)
 iex(web@web-3vmw.us-east1-d.c.firezone-staging.internal)1> {:ok, account} = Domain.Accounts.create_account(%{name: "Firezone", slug: "firezone"})
 {:ok, ...}
 
-iex(web@web-3vmw.us-east1-d.c.firezone-staging.internal)2> {:ok, magic_link_provider} = Domain.Auth.create_provider(account, %{name: "Magic Link", adapter: :email, adapter_config: %{}})
+iex(web@web-3vmw.us-east1-d.c.firezone-staging.internal)2> {:ok, magic_link_provider} = Domain.Auth.create_provider(account, %{name: "One-Time Password", adapter: :email, adapter_config: %{}})
```

Comment:
Should we call it an "email code" or something like that? OTP is an extensive term, which, for example, includes TOTP. I've tried intentionally not to use it because later we might need OTP implementations for 2FA and it will make codebase and terminology ambiguous.

Reply:
Yeah, simply "Email" might work better as the name here, then we can call them "sign in tokens" when referring to the actual token?

Reply:
Yeah, sounds good

Reply:
Updated